src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,65 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+import mate.jdbc.service.impl.DriverServiceImpl;
+import mate.jdbc.service.impl.ManufacturerServiceImpl;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        final ManufacturerService manufacturerService
+                = (ManufacturerServiceImpl) injector.getInstance(ManufacturerService.class);
+        final DriverService driverService
+                = (DriverServiceImpl) injector.getInstance(DriverService.class);
```

Comment:
I'd also recommend making these services the class fields (as you've done with injector)